libtsuba/include/tsuba/TxnContext.h

```diff
@@ -0,0 +1,92 @@
+#ifndef KATANA_LIBTSUBA_TSUBA_TXNCONTEXT_H_
+#define KATANA_LIBTSUBA_TSUBA_TXNCONTEXT_H_
+
+namespace tsuba {
+
+class KATANA_EXPORT TxnContext {
+public:
+  TxnContext(){};
```

Comment:
If you aren't going to implement them differently than the compiler would, you should just omit the default constructor and destructor